pgmpy/estimators/HillClimbSearch.py

```diff
@@ -158,6 +158,8 @@ def estimate(self, start=None, tabu_length=0, max_indegree=None):
 
             if best_operation is None:
                 break
+            if best_score_delta < epsilon:
```

Comment:
@karttikeya I think your changes are fine. @kris-singh I don't think there is a very easy way to do what you are suggesting.